ArduCopter/events.cpp

```diff
@@ -34,7 +34,7 @@ void Copter::failsafe_radio_on_event()
             desired_action = Failsafe_Action_SmartRTL_Land;
             break;
         case FS_THR_ENABLED_ALWAYS_LAND:
-            desired_action = Failsafe_Action_Land;
+            desired_action = Failsafe_Action_None;
```

Comment:
@AkshayMatternet what behavior issue does this change address? It's not note in the PR description

Reply:
This stops the aircraft from triggering the failsafe action, which is, the aircraft not arming and switching to LAND mode every time it gets an Arm command while the aircraft is on the current FS configuration and the RC is turned Off.